rdrf/rdrf/checks/security.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from django.conf import settings
+from django.core.checks import Error, register
+from django.urls import get_resolver, URLPattern, URLResolver
+
+
+@register()
+def url_whitelist_check(app_configs, **kwargs):
+    registered_names = set()
+
+    def get_url_names(patterns, prefix=""):
+        for pattern in patterns:
+            if isinstance(pattern, URLPattern) and pattern.name:
+                registered_names.add(f"{prefix}{pattern.name}")
+            elif isinstance(pattern, URLResolver):
+                get_url_names(pattern.url_patterns, f"{prefix}{pattern.namespace}:" if pattern.namespace else prefix)
+
+    get_url_names(get_resolver().url_patterns)
+
+    with open(os.path.join(settings.WEBAPP_ROOT, "rdrf/checks/url_whitelist.txt"), 'r') as f:
```

Comment:
This approach doesn't really work for the forks, so ideally it should be done in the app settings. However, there are nearly 400 urls so we shouldn't add all of them to a settings file. Maybe the settings file should point to a txt file where urls are listed